Source/WebCore/Modules/indexeddb/IDBActiveDOMObjectInlines.h

```diff
@@ -43,7 +43,10 @@ void IDBActiveDOMObject::performCallbackOnOriginThread(T& object, void (T::*meth
 
     Locker<Lock> lock(m_scriptExecutionContextLock);
 
-    ScriptExecutionContext* context = scriptExecutionContext();
+    // We cannot ref `context` here as it would ref the Document off the main thread,
+    // which is not currently safe. The call to `postCrossThreadTask()` below cannot
+    // destroy the `context` synchronously so the code is safe as is.
+    SUPPRESS_UNCOUNTED_LOCAL auto* context = scriptExecutionContext();
```

Comment:
Hm... this is very unfortunate. I wonder if we can make `ScriptExecutionContext` support thread safe CheckedPtr and deploy that here?